VersionManager.py

```diff
@@ -1,4 +1,4 @@
-#! /usr/bin/env python2
+#! /usr/bin/env python
```

Comment:
I think it would be better if this specified `python3` rather than just `python`